site/en/setup.md

```diff
@@ -11,13 +11,24 @@ OCR-D's software is a modular collection of many projects (called _modules_)
 with many tools per module (called _processors_) that you can combine freely
 to achieve the workflow best suited for OCRing your content.
 
+## System requirements
+
+Minimum system requirements
+* 8 GB RAM (more recommended)
+* min. 100 GB free disk space for local installation 
+  - (min/max similar to Docker?)
```

Comment:
The intention behind the large HDD requirement is too prevent users from provisioning appliances that are too small to hold not only the actual software size (which is indeed comparable to the sizes of Docker) but also models (which can be huge) as well as processing data. With multiple venvs, the space requirements can grow rather quickly, so it is wise to err on the side of "too much HDD"

Reply:
Not to forget ocrd_olena which has really huge demands for temporary disk space during the build process.

Reply:
I've added a (naive) breakdown on why we recommend so much HDD space.